readmeio/api-explorer
# fix(tutorials): solving ux issues within tutorial modal

## 🧰 What's being changed?

- Moving tutorialModal to top level and adding default object to maintain the modal in the dom
- Fixing issue where steps were not reset on reopen due to reference error
- Updating @readme/ui
- Updating tutorial demo

## 🧪 Testing

Enable tutorials via toggle.
- Click out of modal via "x" (css murky here, blacked out) - should remove backdrop on close
- Click into tutorial seconds step, close modal, and reopen - should open on first step.
